vassal-app/src/main/resources/VASSAL/i18n/VASSAL.properties

```diff
@@ -10,11 +10,12 @@ Dialogs.show_details=Show details
 Dialogs.hide_details=Hide details
 Dialogs.out_of_date=Out Of Date Custom Code
 Dialogs.deprecated.later=some later date
-Dialogs.deprecated.message=This module contains custom code that is calling the VASSAL function %1$s. This function is scheduled to be removed from VASSAL after %2$s and this module will cease to function.\n\nPlease check whether there is an updated version of this module. If not, please contact the maintainer of this module and request that it be fixed.
-Dialogs.out_dated_usage=This module contains custom code that is calling into VASSAL in a way that is no longer recommended (%1$s). This will result poor performance and/or bugs.\n\nPlease check whether there is an updated version of this module. If not, please contact the maintainer of this module and request that it be fixed.
+Dialogs.deprecated.message=This module contains custom code that is calling the VASSAL function %1$s. This function is scheduled to be removed from VASSAL after %2$s and this module will cease to function.
+Dialogs.check_for_updated_module=Please check whether there is an updated version of this module. If not, please contact the maintainer of this module and request that it be fixed.
+Dialogs.out_dated_usage=This module contains custom code that is calling into VASSAL in a way that is no longer recommended (%1$s). This will result in poor performance and/or bugs.
 Dialogs.incompatible.title=Incompatible Custom Code
 Dialogs.incompatible.heading=The Custom Code In This Module Should Be Recompiled
-Dialogs.incompatible.message=This module contains custom Java code (%1$s) which was not compiled to be Java 11 compatible. As a result, this module will not run on all versions of Java which VASSAL itself supports.\n\nPlease check whether there is an updated version of this module. If not, please contact the maintainer of this module and request that it be fixed.
+Dialogs.incompatible.message=This module contains custom Java code (%1$s) which was not compiled to be Java 11 compatible. As a result, this module will not run on all versions of Java which VASSAL itself supports.
```

Comment:
Why remove the last bit?

Reply:
Oh, I see. You've moved it to its own key.

Reply:
Yep, same sentence was used in 3 different places